monitoring/rid_qualifier/aircraft_state_replayer.py

```diff
@@ -12,22 +12,24 @@
 import arrow
 import pathlib
 
-from typing import List
+from typing import List, Optional
 from monitoring.rid_qualifier.utils import RIDQualifierTestConfiguration
 
 class TestBuilder():
     ''' A class to setup the test data and create the objects ready to be submitted to the test harness '''
 
-    def __init__(self, test_configuration: RIDQualifierTestConfiguration) -> None:
+    def __init__(
```

Comment:
nit: when breaking a line into multiple lines, the following lines should be indented twice to distinguish the arguments from the code that follows